INTEGRATE.md

```diff
@@ -0,0 +1,33 @@
+# Local integration between Porta/System with Keycloak through Zync
+With these 3 components installed, you will be able to synchronize your applications in Porta with your Keycloak (Red Hat SSO)
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+You can have a keycloak running locally with `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+So that docker container with keycloak will be accessible from your browser in `http://localhost:8080`.
+
+After that, you will need to follow the instructions to [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
+
+## Run Zync
+From the folder where you have Zync installed, run: `ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -p 5000` so you will get zync running in the port 5000.
+
+## Run Porta
+If your porta is already running, stop it, and if you have `spring` running, also do `spring stop`.
+You will need these configurations:
+  - In `config/rolling_updates.yml`, make sure you have `apicast_v2: true` and `apicast_oidc: true`.
```

Comment:
I think we should make those rolling updates default in Porta now so you can remove those instructions

Reply:
@hallelujah Do you mean in the Porta repo making a PR with these configs in `config/examples/rolling_updates.yml` ?

Reply:
No I mean removing the protection of rolling updates on those feature.
They should be default now

Reply:
@hallelujah I've just created this https://issues.jboss.org/browse/THREESCALE-2059
So this is not a blocker to merge this for now and I can do the more urgent tasks first and people can have some instructions to use Zync.